HPXMLtoOpenStudio/resources/EPvalidator.stron

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<sch:schema xmlns:sch="http://purl.oclc.org/dsdl/schematron">
+  <sch:title>HPXML Schematron Validator</sch:title>
+  <sch:ns uri='http://hpxmlonline.com/2019/10' prefix='hpxml_v3'/>
```

Comment:
I suggest just using prefix='h' instead, since it's going to show up so often in the file.